### PR TITLE
added in accept button to accept invitation

### PR DESCRIPTION
**Pull request by ninagong**

### CODE REVIEW

File: backend/src/database.ts [hunk 1]
```diff
@@ -72,7 +72,7 @@ export async function createMember(
 
   return resp.data;
 }
-
+//add comment
```
Comment (ninagong) [inline backend/src/database.ts:75]: rm

File: backend/src/database.ts [hunk 2]
```diff
@@ -187,3 +187,30 @@ export async function deleteGroup(groupId: string): Promise<boolean> {
     return false;
   }
 }
+
+//Function to update the accepted status of a group in the database
+export async function acceptInvitedGroup(email, groupID) {
```
Comment (ninagong) [inline backend/src/database.ts]: add type param

File: backend/src/database.ts [hunk 3]
```diff
@@ -187,3 +187,30 @@ export async function deleteGroup(groupId: string): Promise<boolean> {
...
+ 
+  try {
+
+    // Update the 'accepted' field of the member with the specified email and group ID
+    const resp = await supabase
+      .from('members')
+      .update({ accepted: true })
+      .eq('email', email)
+      .eq('group_id', groupID);
+    console.log("data is ",resp)
```
Comment (ninagong) [inline backend/src/database.ts]: rm logs

File: frontend/src/components/GroupList.tsx [hunk 4]
```diff
@@ -69,6 +69,40 @@ export default function GroupList(props: { session: Session | null }) {
         console.error("Error deleting group:", error);
       });
   };
+  const handleDeclineInvitation (groupToDecline) => {
+    const groupId = groupToAccept.id;
```
Comment (ninagong) [inline frontend/src/components/GroupList.tsx]: add comment saying TODO(nina)

File: backend/src/server.ts [hunk 5]
```diff
@@ -163,3 +164,27 @@ app.delete("/api/groups/:id", async (req, res) => {
 app.listen(PORT, () => {
   console.log(`Server is running on http://localhost:${PORT}`);
 });
+
+app.put("/api/groups/:id", async (req, res) => {
```
Comment (ninagong) [inline backend/src/server.ts]: api/groups/:id

api/accept_invite/:groupId

File: backend/src/database.ts (hunk 1 above)
Comment (ninagong) [reply backend/src/database.ts:75]: did

**Review by Winxton — APPROVED** [review]
LGTM

**Comment by ninagong** [comment]
LGBT

**Pull request merged**